### ISSUE TITLE
configuration: move configuration for LaxarJS paths to the application configuration

### ISSUE THREAD

**Issue by alex3683** [LaxarJS/laxar#264]
Affects `lib/loaders/paths`: Currently all paths are configured by using RequireJS configuration. Since we want to get rid of AMD, the configuration has to take place somewhere else. One candidate would be `application/application.js`, but it may be harmful to put "setup"-configuration into the same place as the runtime configuration (e.g. an administrator may change the log level of the application whenever it is necessary, but he actually never needs to change the internal directory layout settings and should probably be prevented from doing it accidentally).
@x1B : Your opinion?

**Comment by x1B** [comment]
As long as there are suitable defaults, I think it would be alright to put this information into the application config. These paths are probably more deployment-specific than some of the configuration we already have.

**Comment by alex3683** [comment]
Implemented on `master` (`v2.0.0`)

### Breaking Change
`lib/loaders/paths` is no longer available. Each internal service requiring a specific path, needs to add it as argument to its `create` function.
Adjustment of these paths is no longer an AMD/RequireJS configuration, but an application configuration (`application/application.js`). The default values can be overwritten with specific paths under the key `paths`. These values can be set (the LaxarJS default values are on the right side):
```
paths: {
   product: '',
   themes: 'includes/themes',
   layouts: 'application/layouts',
   controls: 'includes/controls',
   widgets: 'includes/widgets',
   pages: 'application/pages',
   flowJson: 'application/flow/flow.json',
   defaultTheme: 'includes/themes/default.theme'
}
